src/config.rs

```diff
@@ -27,6 +27,27 @@ pub struct Config {
     pub(crate) conf_dirs: Vec<PathBuf>,
 }
 
+impl Default for Config {
+    /// Default configuration.
+    fn default() -> Self {
+        Self {
+            tab_stop: 4,
+            quit_times: 2,
+            message_duration: Duration::from_secs(3),
+            show_line_num: true,
+            conf_dirs: vec![
+                Ok(String::from("/etc/kibi")),
+                env::var("XDG_CONFIG_HOME").map(|d| d + "/kibi"),
+                env::var("HOME").map(|d| d + "/.config/kibi"),
+            ]
+            .into_iter()
+            .filter_map(Result::ok)
+            .map(PathBuf::from)
+            .collect(),
+        }
```

Comment:
Nit: you can save 3 LOCs like this:
```suggestion
        let conf_dirs = vec![
            Ok(String::from("/etc/kibi")),
            env::var("XDG_CONFIG_HOME").map(|d| d + "/kibi"),
            env::var("HOME").map(|d| d + "/.config/kibi"),
        ];
        Self {
            tab_stop: 4,
            quit_times: 2,
            message_duration: Duration::from_secs(3),
            show_line_num: true,
            conf_dirs: conf_dirs.into_iter().filter_map(Result::ok).map(PathBuf::from).collect(),
        }
```

Reply:
Nice